### DISCUSSION TITLE
Is Sysmon rolled into cutom Elastic Agent?

### DISCUSSION THREAD

**Discussion by innovate-support** [Security-Onion-Solutions/securityonion#11974] [2.4] [answered]
### Version

2.4.30

### Installation Method

Network installation on Debian

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

other (please provide detail below)

### Hardware Specs

Exceeds minimum requirements

### CPU

6

### RAM

16

### Storage for /

265

### Storage for /nsm

265

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I've migrated my Endpoints from 2.3 Wazuh to 2.4 Elastic Agent and now am adding a few new Endpoints to the Fleet.  The 2.3 Wazuh method required that I install Sysmon seperately and edit the Wazuh config to gather those alerts, but I've read a few places that Elastic Agent may actually have Sysmon included.  Does the custom SO Elastic Agent have Sysmon rolled into it, or do I need to install Sysmon seperately?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment, marked as answer]
The Elastic Agent does not include sysmon but it does provide much of the same coverage as sysmon. You might want to try Elastic Agent by itself and see if it provides the coverage that you need. If you need additional coverage, then you can install sysmon separately.
